exercises/concept/annalyns-infiltration/.docs/hints.md

```diff
@@ -0,0 +1,11 @@
+# Hints
+
+## General
+
+- There are three boolean operators to work with boolean values: `not`, `&&`, and `||`.
+- Multiple operators can be combined in a single expression.
+
+## 1. Check if a fast attack can be made
+
+- The boolean operators can also be applied to boolean parameters.
+- Unlike many other languages, boolean negation is _not_ done with the `!` operator.
```

Comment:
Leaving the same for now as it might also work for pacman-rules. Not much much different can be said about booleans.